From 20362b6aefa3aeeae34fb869ad017bd7c87e2425 Mon Sep 17 00:00:00 2001
From: Trpl7ca <52380252+Trpl7ca@users.noreply.github.com>
Date: Mon, 16 Jun 2025 14:46:04 -0700
Subject: [PATCH] Update GlucoseBasedApplicationFactorStrategy.swift

---
 Loop/Models/GlucoseBasedApplicationFactorStrategy.swift | 6 +++---
 1 file changed, 3 insertions(+), 3 deletions(-)

diff --git a/Loop/Loop/Models/GlucoseBasedApplicationFactorStrategy.swift b/Loop/Loop/Models/GlucoseBasedApplicationFactorStrategy.swift
index 3be01d553..7c56376cd 100644
--- a/Loop/Loop/Models/GlucoseBasedApplicationFactorStrategy.swift
+++ b/Loop/Loop/Models/GlucoseBasedApplicationFactorStrategy.swift
@@ -13,11 +13,11 @@ import LoopCore
 
struct GlucoseBasedApplicationFactorStrategy: ApplicationFactorStrategy {
+    static let minPartialApplicationFactor = 0.50 // min fraction of correction when glucose > minGlucoseSlidingScale
-    static let maxPartialApplicationFactor = 1.50 // max fraction of correction when glucose > maxGlucoseSlidingScale
+    static let maxPartialApplicationFactor = 1.25 // max fraction of correction when glucose > maxGlucoseSlidingScale
     // set minGlucoseSlidingScale based on user setting for correction range
     // use mg/dL for calculations
-    static let minGlucoseDeltaSlidingScale = 10.0 // mg/dL
-    static let maxGlucoseSlidingScale = 200.0 // mg/dL
+    static let minGlucoseDeltaSlidingScale = 5.0 // mg/dL
+    static let maxGlucoseSlidingScale = 160.0 // mg/dL
 
     func calculateDosingFactor(
         for glucose: HKQuantity,
